williamntlam/rate-my-rental
# Feature/user crud operations

### Problem
There were currently no implemented API endpoints for User data.

### Solution
- API endpoints were created for USER data and operations.

### Key Changes
- Updated code in the src/routes/userRoutes.ts file.

### Testing Instructions
1. Pull the branch and run `npm install` to update dependencies.
2. Run `npm test` to ensure all tests pass.
3. Manually test the login functionality by entering various passwords, including edge cases like special characters and empty fields.

### Potential Impact
User data can now be stored in the database with API endpoints.

### Feedback Request
I’m particularly interested in feedback on the approach taken in userRoutes.ts - Please let me know!
